mirrord/cli/src/execution/windows.rs

```diff
@@ -0,0 +1,171 @@
+// mod common;
```

Comment:
this deserves a thorough review before being merged, i believe. shouldw e agree, i'll do it once the PR is ready for review.